crates/matrix-sdk-base/src/client.rs

```diff
@@ -56,6 +56,9 @@ use tokio::sync::RwLock;
 use tokio::sync::RwLockReadGuard;
 use tracing::{debug, info, instrument, trace, warn};
 
+#[cfg(feature = "e2e-encryption")]
+#[cfg(feature = "experimental-sliding-sync")]
```

Comment:
```suggestion
#[cfg(all(feature = "e2e-encryption", feature = "experimental-sliding-sync"))]
```

Reply:
Done in f564fef5642b24bfa82cbf7e16734e58129a9612